### PR TITLE
Feat/start questionnaire bylaws pages

### LINKED ISSUE #13: Three Pages: Start, Questions, Bylaws
## Value statement - a requirement, not a solution
As a user
I need a start page, a question page and a bylaws page
so that I know what to do, answer questions and see the bylaws separated.

## Description (Text, Scribble, Wireframe, Design)
Bylaws-2020-09-18

## Acceptance criteria
- [x] user can click a button on the start page (path: /) to start the questions
- [x] user can click a button on the questions page (path: /questions) to go to the generated bylaws 
- [x] user can click a button on the bylaws text page (path: /bylawstext) to go back to the questions 

## Tasks
- [x] create feature branch
- [x] add Router
- [x] path '/start' for start page
- [x] path '/questionnaire' for questionnaire page
- [x] path '/bylawstext' for bylaws text page
- [x] start-button
- [x] bylaws-button
- [x] back to questions-button
- [x] update cypress tests
- [x] add cypress tests for start and bylaws text page

## Size
medium

### PR DESCRIPTION
**Pull request by carolinwagner**
Please review this story - I added React Router and some navigation buttons :-)

Preview: https://capstone-project-git-feat-start-questionnaire-bylaws-pages.carolinwagner.vercel.app

Closes #13

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,17 +1,28 @@
...
+import BylawsText from './BylawsText'
 import QuestionsForm from './QuestionsForm'
+import Start from './Start'
 
 function App() {
   const [answers, setAnswers] = useState([])
 
   return (
-    <>
+    <Router>
```
Comment (maximiliangustmann) [inline src/App.js:12]: I would move the Router to your index.js:
`<Router>
      <App />
    </Router>`

**Review by maximiliangustmann — COMMENTED** [review]
Very nice usage of react-router!

File: src/App.js (hunk 1 above)
Comment (carolinwagner) [reply src/App.js]: Thanks for the review :-) I moved the Router to the index.js

**Comment by karstenlenz** [comment]
QA results:

The navigation works!  I noticed some things when filling out the questionnaire and navigating:

- questions 10,19,24: if you enter a wrong number, the error message is still in English.(my browser language is set to English, may be related)
- questions 10,19,24: it is easy to enter a wrong number because the user may not know the requirements (such as needing at least 7 members to sign). It would be nice to have a hint in the question text (min.7). Then it is less frustrating than getting too many error messages.
- after filling out the form and submitting, the user needs to manually scroll to the top of the list. The same happens when you press "back to the questions". It would be nicer if the pages would always started at the top.
- on Android/Chrome: The behavior is a little bit weird. On some inputs the browser doesn't jump to the next input, but skips some. For example: After filling out question 3 it jumps to question 10.  After filling out question 10, the browser jumps to question 19. From 19 it jumps to 24. It seems to have something to do with the input types (text, checkbox, etc.)
- There is no limit on the founding members. I'm not sure if that might make sense not to have a million founding members.

**Comment by carolinwagner** [comment]
Thank you for your review! 

>     * questions 10,19,24: if you enter a wrong number, the error message is still in English.(my browser language is set to English, may be related)
Yes, this is due to the native browser validation - I'm afraid I cannot change that

>     * questions 10,19,24: it is easy to enter a wrong number because the user may not know the requirements (such as needing at least 7 members to sign). It would be nice to have a hint in the question text (min.7). Then it is less frustrating than getting too many error messages.
✅ 
>     * after filling out the form and submitting, the user needs to manually scroll to the top of the list. The same happens when you press "back to the questions". It would be nicer if the pages would always started at the top.
✅ 

>     * on Android/Chrome: The behavior is a little bit weird. On some inputs the browser doesn't jump to the next input, but skips some. For example: After filling out question 3 it jumps to question 10.  After filling out question 10, the browser jumps to question 19. From 19 it jumps to 24. It seems to have something to do with the input types (text, checkbox, etc.)
working on that one - thanks for the input, I didn't even notice! 

>     * There is no limit on the founding members. I'm not sure if that might make sense not to have a million founding members.
As a developer it is not my part to decide how many founding members is the max - since there is no legal maximum, only a legal minimum. Any number would be random.

**Pull request merged**